README.md

```diff
@@ -70,8 +76,19 @@ The bootstrap.sh can be invoked multiple times but it has a status control to en
 of the libraries is loaded only once. This is to prevent potential issues with collections 
 that are not supposed to be loaded many times.
 
-An example of this is the _container_engine_ library, where the selected container engine
-is **set only once the first command using the library helper function `container_engine_cmd` is used**.
+An example of this is the _container_ library, where the selected container engine
+is **set only once the first command using the library helper function `cicd_tools::container::cmd` is used**.
+
+Each of the libraries will export to the shell sourcing the bootstrap script the helper functions.
+These functions are all namespaced, meaning the names follow the naming format:
+
+```
+cicd_tools::library::function
```

Comment:
Something to consider: Should the root namespace just be shortened to `cicd` for easier typing? Might make the code a bit easier to read too.

Reply:
I decided to rename it to `cicd`  - thanks for make me reconsider this :+1: